tsconfig.json

```diff
@@ -1,17 +1,21 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es5",
-        "noImplicitAny": false,
-        "rootDir": "./src",
-        "outDir": "./release",
-        "removeComments": true,
-        "experimentalDecorators": true,
-        "emitDecoratorMetadata": true
-    },
-    "exclude": [
-        "node_modules",
-        "typings",
-        "release"
-    ]
-}
\ No newline at end of file
+  "compilerOptions": {
+    "declaration": true,
+    "declarationDir": "dist/types",
+    "outDir": "dist/commonjs",
+    "lib": ["es2017", "dom"],
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": false,
+    "noUnusedParameters": false,
+    "noUnusedLocals": true,
+    "removeComments": true,
+    "sourceMap": true,
+    "strictNullChecks": false,
+    "target": "es5",
+    "module": "es2015",
+    "moduleResolution": "node"
+  },
+  "include": ["src"]
```

Comment:
Is there an option to exclude spec files building types?

Reply:
I don't know, maybe it accepts some negation. I don't think it is that crucial though. I would like to exclude it altogether but WebPack prints warning without it, perhaps with rollup now it would work ?